config/locales/en.yml

```diff
@@ -38,15 +38,22 @@ en:
       short: "%m/%d"
       time_12_hour: "%l:%M %p %Z"
       day_and_date: "%A, %m/%d"
-  # validation error messages
+  # validation error messages https://guides.rubyonrails.org/i18n.html#error-message-interpolation
   activerecord:
     errors:
+      messages:
+        blank: Can't be blank
+        not_http_url: Not a valid URL
       models:
         episode:
           attributes:
             segment_count:
               greater_than: Can't be negative
               less_than_or_equal_to: Enter a number less than %{count}
+        podcast:
+          attributes:
+            payment_pointer:
+              invalid: Payment pointers must have the format $prx.wallet.example/abcd1234
```

Comment:
I haven't translated all error messages - but I think they're looking okay?  Maybe we'll identify more to add here, during QA.